bigchaindb/commands/bigchain.py

```diff
@@ -88,7 +88,7 @@ def run_drop(args):
 
 def run_start(args):
     """Start the processes to run the node"""
-    run_configure(args, skip_if_exists=True)
+    # run_configure(args, skip_if_exists=True)
```

Comment:
@ttmc: please note that I disabled the "autoconfiguration" feature. We might need to update the docs.